### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by donizer**
- [DEMO LINK](https://donizer.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://donizer.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="index.html">
+        <img src="./images/logo.png" alt="logo" >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">apple</a>
+          </li>
```
Comment (VitaliyBodnar) [inline src/index.html:23]: please add spaces between the `<li>` blocks as it will make a code cleaner

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,83 @@
...
+  all: unset;
+  list-style-type: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  /* height: 60px; */
```
Comment (VitaliyBodnar) [inline src/style.css:27]: remove all comments

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,83 @@
...
+}
+
+.nav__link.is-active::after {
+  content: "";
+  position: absolute;
+  background-color: var(--accent-color);
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
```
Comment (VitaliyBodnar) [inline src/style.css:81]: Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,83 @@
...
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+  top: 0;
+  bottom: 0;
```
Comment (VitaliyBodnar) [inline src/style.css:49]: top and bottom are not working with `position: static;`

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (donizer) [reply src/style.css:81]: like this?
```
  content: "";
  position: absolute;
  bottom: 0;
  left: 0;
  right: 0;
  height: 4px;
  border-radius: 8px;
  background-color: var(--accent-color);
```

**Comment by donizer** [comment]
I am not sure why test fails with code 1.

[Here](https://donizer.github.io/layout_moyo-header/) and [in report](https://donizer.github.io/layout_moyo-header/report/html_report/) all seems to be working fine.

**Comment by donizer** [comment]
Just completed next task without issues. Maybe i should just start over this task?

File: src/style.css (hunk 3 above)
Comment (VitaliyBodnar) [reply src/style.css]: yep )

**Review by VitaliyBodnar — APPROVED** [review]
looks like there are some issues with the deploying scripts, it's not your fault.
Html reports  is good, so approving.

**Pull request closed without merging**